nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/FlowController.java

```diff
@@ -3279,14 +3256,14 @@ public synchronized void onLeaderElection() {
         });
     }
 
+
     /**
      * Sets whether this instance is clustered. Clustered means that a node is either connected or trying to connect to the cluster.
      *
      * @param clustered true if clustered
      * @param clusterInstanceId if clustered is true, indicates the InstanceID of the Cluster Manager
-     * @param clusterManagerDn the DN of the NCM
      */
-    public void setClustered(final boolean clustered, final String clusterInstanceId, final String clusterManagerDn) {
+    public void setClustered(final boolean clustered, final String clusterInstanceId) {
```

Comment:
@olegz - I think it is accurate that this is called only from other classes within the same package (I believe only StandardFlowService, in fact). And if that is the case then yes, I have no problem with making it package-private. Feel free to do so on merge.